### ISSUE TITLE
Agenda: Community Council (CC), 20 December 2022

### ISSUE THREAD

**Issue by cekickafa** [grincc/agenda#75]
Solicit suggestions for agenda items for the Governance meeting to be held on Tuesday December 20 @ **10:00 UTC** in grincoin#general channel on [Keybase](https://keybase.io/team/grincoin#general). Please comment to provide topics or suggestions.
(Meeting can be followed on @keybase-ro channel - GRIN community [discord server](https://discord.gg/5p4vCQY9km) via GRIN discord bridge).

## To Do List.

1) Contact @walkbackgod and @marekyggdrasil about Python CFFI secp256k1-zkp wrapper.

## Proposed Agenda
- Request for funding @davidtavarez [January-April 2023](https://forum.grin.mw/t/request-for-funding-davidtavarez-january-april-2023/10205).

- Request for funding **@cobragrin** [Dec-March 2023](https://forum.grin.mw/t/request-for-funding-cobragrin-december-2022-march-2023/10208).
- Request for funding **@jaw709** [Jan-April 2023](https://forum.grin.mw/t/request-for-funding-groundskeeper-satoshocrat-jan-apr-2023/10211).
- Request for @navie [funding Dec 22- Q1 2023](https://forum.grin.mw/t/request-for-funding-navie-dec-2022-q1-2023/10210).
-  Wallet issues; [mimblewimble/grin-wallet](https://github.com/mimblewimble/grin-wallet/issues).
   - [owner api](https://github.com/mimblewimble/grin-wallet/issues/665).

**Comment by davidtavarez** [comment]
I want to make it easy for people to develop new things on top of Grin++ now and in the future, but that requires a robust API. And that’s what I want to continue to focus on. The current API is very UI-bound at the moment, and this should be improved. I have been working on improving the API using as a guide the API in Rust, the RFC and adding the changes that I think are important. In order to make sure things are working fine, I will bring to life the CLI version of Grin++, this CLI version should be using the new APIs and it should have a basic compatibility with the Rust API. 

Recently Satoshocrat managed to compile the master branch for a Raspberry Pi4, this is a huge step, and simplifies the efforts on increasing portability. I would love to make possible to users to run their Grin++ wallets on their Pi4, and a CLI version seems to be more reasonable to me.

**Comment by davidtavarez** [comment]
### Regarding the @navie funding.

I support it as long as they can prove that the miners are mining the test network,

### Regarding the tasks of the Groundkeepers.

I understand that @cekickafa is taking quite a load and that the load could be split between the two Groundkeepers, as it was supposed to be. Besides, the month is almost over, it would make more sense to do it from January to April.

The main tasks are as follows (and my comments on each of them):

1) Administrative tasks.
   - Agenda maintenance.
   - Transcribe meeting notes and summary.
   - Decision log updates.
   - Tracking Task List.
 
I think @cekickafa is doing a great job on this and I support him continuing to do the same).

2) Writing & Publishment of Developments, upgrades and news.
   - Grinpost newsletter.
   - Grinpost medium.

So far the work here is OK. I would like to see more collaboration between @jaw709 and @cekickafa. I want to express again that we must be very careful with what is published in the newsletter. One could write the draft and the other could be the editor, because it is important to be very precise with what you write. The content should be better reviewed before publishing, and I am not sure if this is being done.

I would like to see how @jaw709 performs managing the next two main tasks:

3) Finance PR Contributing to grincc/finance PR for transparency and accountability.
   - Prepare account & financial reports.
   - Spending and payment logs
   - Quarterly transparency reports bookkeeping and publish.
   - Management and Tracking payments

4) Tracking, calculating and making sure the draft are created.

Drafting the transaction should be done by one of the GrinCC members, not by any of the Groundkeepers.

### Grin Newspaper format.

I support transforming the bi-weekly blog post into a Newspaper format, with above-the-fold headlines, catchy pieces with engaging content, and visually, as a website. This could help us to include technical articles and expand what is published, and consequently, we would improve the content. I would like to @jaw709 doing this.

**Comment by cekickafa** [comment]
>So far the work here is OK. I would like to see more collaboration between @jaw709 and @cekickafa. I want to express again that we must be very careful with what is published in the newsletter. One could write the draft and the other could be the editor, because it is important to be very precise with what you write. The content should be better reviewed before publishing, and I am not sure if this is being done.

We tried this and it is not working much. I am doing the Newsletter since 2021 and According to the statistics, my newsletter is highly effective. I began it in 2021, when the official one(Lehnbergs) was suspended, and i continued publishing without fund request etc. I will not let it go regardless. 

>I would like to see how @jaw709 performs managing the next two main tasks:

    Finance PR Contributing to grincc/finance PR for transparency and accountability.
        Prepare account & financial reports.
        Spending and payment logs
        Quarterly transparency reports bookkeeping and publish.
        Management and Tracking payments

    Tracking, calculating and making sure the draft are created.

I performed all those well. 

>Drafting the transaction should be done by one of the GrinCC members, not by any of the Groundkeepers.   

_Payment tracking duty and drafting transactions already added and voted.
_https://github.com/cekickafa/agenda/blob/main/decision_log.md #33_ 


>I support transforming the bi-weekly blog post into a Newspaper format, with above-the-fold headlines, catchy pieces with engaging content, and visually, as a website. This could help us to include technical articles and expand what is published, and consequently, we would improve the content. I would like to support @jaw709 doing this.

We already have this. Contribute **[Grincc.mw](https://grincc.mw/)**

**Comment by jaw709** [comment]
Providing some follow-up comments in regards to my funding request since I will not be available 5 am EST.

Right off, I want to thank CKafa for his effort with the Newsletter and his diligence in ensuring most note-worthy events/progress are covered. In my opinion however, there is room for improvement in both display of content and quality. To me, the current newsletter comes off as a "blog," which is fine, but it doesn't convey the level of professionalism I think we can achieve with a fuller-format Newspaper; it also requires additional oversight for grammar, phrasing, diction etc. This is partly why I have the role of assistant groundskeeper now.

The Newspaper will be also open to community content hosted on the site (wordpress, squarespace etc.) and I believe will offer a more dynamic, visual appeal in the layout, headlines, "articles" as opposed to the newsletter blog format. Just some background on my education I suppose:  I did take Fundamentals to Journalism back at school, so I am familiar with "Inverted Pyramid" organization, how to write compelling ledes, and I have a general appreciation for language. 

I do believe there is active interest in this idea across the Grin community, but if I am way off base, then I will be happy to be wrong too. The Groundskeepers' duties have always been at the Council's discretion, and I feel it's about aligning talents and passion with the best outcome/betterment of the Grin project. 

I understand there is some contest with the "full" Groundskeeper compensation I am requesting, but I will say my contribution will not stop at the Newspaper:  creating the website, writing/sourcing pieces. editing content etc. But I am also open to undertaking as @davidtavarez mentioned above, the financial documentation and other technical tasks. I ask this all to be taken into consideration, and if the amount is still deemed superfluous, I am open to discussion. At the end of the day, I am exploring dormant passions while trying to find a symbiotic relationship with meaningful work and life at large. 


Thank you,

Satoshocrat

**Comment by davidtavarez** [comment]
> We already have this. Contribute **[Grincc.mw](https://grincc.mw/)**

You are right, grincc.mw can be use for this purpose, the code is open source and jekyll can be use for this.

> Payment tracking duty and drafting transactions already added and voted.

Maybe we are talking about two different things here. Access to the key must be granted by the proper procedure, when I say "drafting" the transactions I mean literally create the transaction file with one initial signature added.
